requirements_all.txt

```diff
@@ -383,6 +383,9 @@ pyenvisalink==1.9
 # homeassistant.components.ifttt
 pyfttt==0.3
 
+# homeassistant.components.remote.harmony
```

Comment:
not sure what happened here, i set it back to match the other requirements files